docs/lifecycle.md

```diff
@@ -108,6 +128,23 @@ See AWS specs for details on the protocol format.
 Those calls essentially manage bucket attributes related to lifecycle
 behavior, which are stored as part of bucket metadata.
 
+#### Transition Policies support in CloudServer
```

Comment:
looks like you're going for non-proper nouns not being capitalized in the #### sections, so
Policies -> policies